gdeep/trainer/trainer.py

```diff
@@ -54,7 +54,7 @@
         pass
     print("Using TPU!")
 except ModuleNotFoundError:
-    print("No TPUs...")
+    pass
```

Comment:
Why?

Reply:
It's still WIP; I'll let you know once it's ready to review.